keyboards/enviousdesign/delirium/rgb/info.json

```diff
@@ -0,0 +1,239 @@
+            {"matrix": [3, 14], "x": 175, "y": 29, "flags": 4},
+            {"matrix": [2, 15], "x": 198, "y": 29, "flags": 4},
+            {"matrix": [2, 16], "x": 211, "y": 29, "flags": 4},
+            {"matrix": [2, 17], "x": 224, "y": 29, "flags": 4},
+            {"matrix": [3, 0], "x": 5, "y": 41, "flags": 8},
+            {"matrix": [3, 2], "x": 23, "y": 41, "flags": 4},
+            {"matrix": [3, 3], "x": 36, "y": 41, "flags": 4},
+            {"matrix": [3, 4], "x": 49, "y": 41, "flags": 4},
+            {"matrix": [3, 5], "x": 62, "y": 41, "flags": 4},
+            {"matrix": [3, 6], "x": 75, "y": 41, "flags": 4},
+            {"matrix": [3, 7], "x": 88, "y": 41, "flags": 4},
+            {"matrix": [3, 8], "x": 101, "y": 41, "flags": 4},
+            {"matrix": [3, 9], "x": 114, "y": 41, "flags": 4},
+            {"matrix": [3, 10], "x": 127, "y": 41, "flags": 4},
+            {"matrix": [3, 11], "x": 140, "y": 41, "flags": 4},
+            {"matrix": [3, 12], "x": 153, "y": 41, "flags": 4},
+            {"matrix": [3, 13], "x": 166, "y": 41, "flags": 4},
+            {"matrix": [4, 0], "x": 2, "y": 52, "flags": 4},
+            {"matrix": [4, 1], "x": 16, "y": 52, "flags": 4},
+            {"matrix": [4, 2], "x": 29, "y": 52, "flags": 4},
+            {"matrix": [4, 3], "x": 42, "y": 52, "flags": 4},
+            {"matrix": [4, 4], "x": 55, "y": 52, "flags": 4},
+            {"matrix": [4, 5], "x": 68, "y": 52, "flags": 4},
+            {"matrix": [4, 6], "x": 81, "y": 52, "flags": 4},
+            {"matrix": [4, 7], "x": 94, "y": 52, "flags": 4},
+            {"matrix": [4, 8], "x": 107, "y": 52, "flags": 4},
+            {"matrix": [4, 9], "x": 120, "y": 52, "flags": 4},
+            {"matrix": [4, 10], "x": 133, "y": 52, "flags": 4},
+            {"matrix": [4, 11], "x": 146, "y": 52, "flags": 4},
+            {"matrix": [4, 13], "x": 170, "y": 52, "flags": 4},
+            {"matrix": [4, 16], "x": 211, "y": 52, "flags": 4},
+            {"matrix": [5, 0], "x": 2, "y": 64, "flags": 4},
+            {"matrix": [5, 1], "x": 18, "y": 64, "flags": 4},
+            {"matrix": [5, 3], "x": 34, "y": 64, "flags": 4},
+            {"matrix": [5, 6], "x": 83, "y": 64, "flags": 4},
+            {"matrix": [5, 10], "x": 131, "y": 64, "flags": 4},
+            {"matrix": [5, 11], "x": 148, "y": 64, "flags": 4},
+            {"matrix": [5, 13], "x": 164, "y": 64, "flags": 4},
+            {"matrix": [5, 14], "x": 180, "y": 64, "flags": 4},
+            {"matrix": [5, 15], "x": 198, "y": 64, "flags": 4},
+            {"matrix": [5, 16], "x": 211, "y": 64, "flags": 4},
+            {"matrix": [5, 17], "x": 224, "y": 64, "flags": 4}
+        ]
+    },
+    "url": "http://envious.design",
+    "usb": {
+        "device_version": "1.0.0",
+        "pid": "0xA400",
+        "vid": "0xE739"
+    },
+    "ws2812": {
+        "driver": "vendor",
+        "pin": "GP0"
+    },
+    "community_layouts": ["tkl_iso"],
+    "layouts": {
```

Comment:
does this version of delirium have a `tkl_ansi` layout?

Reply:
This version does not have ANSI support.